### ISSUE TITLE
SPDX format does not include package release on `versionInfo`

### ISSUE THREAD

**Issue by paleloser** [aquasecurity/trivy#3892]
## Description
In versions < `0.38.0`, executing `trivy image --format spdx-json` returned the package release on the `versionInfo` field:

```sh
$ trivy_0.37.3_macOS-64bit/trivy image --format spdx-json bitnami/wordpress:latest > report.json
[...]
$ cat report.json | grep "2.36.1-8+deb11u1"                                           
				"PkgID: libsmartcols1@2.36.1-8+deb11u1",
					"referenceLocator": "pkg:deb/debian/libsmartcols1@2.36.1-8+deb11u1?distro=debian-11.6",
			"sourceInfo": "built package from: util-linux 2.36.1-8+deb11u1",
			"versionInfo": "2.36.1-8+deb11u1"
```
But updating to >= `0.38.0` we lost this info:

```sh
$ trivy_0.38.0_macOS-64bit/trivy image --format spdx-json bitnami/wordpress:latest > report.json
[...]
$ cat report.json | grep "2.36.1-8+deb11u1"                                                     
				"PkgID: libsmartcols1@2.36.1-8+deb11u1",
					"referenceLocator": "pkg:deb/debian/libsmartcols1@2.36.1-8+deb11u1?distro=debian-11.6",
			"sourceInfo": "built package from: util-linux 2.36.1-8+deb11u1",
```

This makes hard to clearly identify which release is installed, as it requires parsing other fields such as `referenceLocator`, when it wasn't necessary in the beginning.

## What did you expect to happen?
One would expect to keep getting the full version data through the `versionInfo` field of the `package` object.

## What happened instead?
Only the package version, without release info, is returned.

## Output of run with `-debug`:

```sh
$ trivy image --format spdx-json bitnami/wordpress:latest --output report.json --debug          
2023-03-23T10:48:13.577+0100	DEBUG	["cyclonedx" "spdx" "spdx-json" "github"] automatically enables '--list-all-pkgs'.
2023-03-23T10:48:13.578+0100	DEBUG	Severities: ["UNKNOWN" "LOW" "MEDIUM" "HIGH" "CRITICAL"]
2023-03-23T10:48:13.578+0100	INFO	"--format spdx" and "--format spdx-json" disable security scanning
2023-03-23T10:48:13.620+0100	DEBUG	cache dir:  /Users/pcaraballo/Library/Caches/trivy
2023-03-23T10:48:16.468+0100	DEBUG	Image ID: sha256:67231a0890932d51c1b033287e837ed9c8e44e3610d66f9845cdabdd678a357a
2023-03-23T10:48:16.468+0100	DEBUG	Diff IDs: [sha256:e1a6d9bfd10a5136de94fd475890ec7d9906512024d497bd364b48144c7d574e]
2023-03-23T10:48:16.468+0100	DEBUG	Base Layers: []
```

## Output of `trivy -v`:

```
$ trivy -v
Version: 0.38.3
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-03-23 06:07:45.782507817 +0000 UTC
  NextUpdate: 2023-03-23 12:07:45.782507417 +0000 UTC
  DownloadedAt: 2023-03-23 08:46:58.960025 +0000 UTC
```

## Additional details (base image name, container registry info...):
One weird thing, is that executing the standard output with the latest version (`0.38.3`) returns the full version info:
```sh
$ trivy image bitnami/wordpress:latest | grep "2.36.1-8+deb11u1"        
[...]
│ libsmartcols1         │ CVE-2022-0563    │          │ 2.36.1-8+deb11u1             │               │ util-linux: partial disclosure of arbitrary files in chfn    │
```

If it helps, I saw there were some changes that may be related to this in #3679.

**Comment by knqyf263** [comment]
Thanks for reporting. Yes, #3679 might cause regression.

**Comment by DmitriyLewen** [comment]
Hello @paleloser 

We fixed this problem in #3896 

We will include these changes in [next release](https://github.com/aquasecurity/trivy/milestone/15).
Until then, you can use [canary build](https://github.com/aquasecurity/trivy#canary-builds) if that suits you.

Regards, Dmitriy
